roles/provision-unifiedpush-apb/tasks/main.yml

```diff
@@ -131,6 +139,22 @@
           - "-i"
           - "-c"
           - 'wget http://localhost:8080/ag-push '
+        containers:
```

Comment:
I think we're missing the part for telling kubernetes what serviceaccount to run our pod as

e.g. https://github.com/aerogearcatalog/metrics-apb/blob/master/roles/provision-metrics-apb/tasks/provision-prometheus.yml#L36

Reply:
resolved